web/src/settings_components.ts

```diff
@@ -301,6 +301,18 @@ function get_jitsi_server_url_setting_value(
     return JSON.stringify($custom_input_elem.val());
 }
 
+export function update_custom_time_limit_minute_text($input: JQuery): void {
+    const $minutes_text = $input.parent().find(".time-unit-text");
+    if ($minutes_text.length === 0) {
+        return;
+    }
+    if (Number($input.val()) === 1) {
+        $minutes_text.text($t({defaultMessage: "minute"}));
+    } else {
+        $minutes_text.text($t({defaultMessage: "minutes"}));
+    }
```

Comment:
Similar to the handlebar templates, it would make sense to use the[ICU MessageFormat](https://formatjs.io/docs/intl-messageformat) here instead of the if/else loop.